### DISCUSSION TITLE
No Docker manifest for linux/arm/v7

### DISCUSSION THREAD

**Discussion by rymiller11** [ep1cman/unifi-protect-backup#124] [Q&A]
Hello @ep1cman :  I have a problem when trying to run UPB with docker on my RPi 3B.  There is no version for this version within Docker and therefore it will not install/run.  Could you add this to the repository?
Thanks!

**Comment by ep1cman** [comment]
As can be seen on the OS/Arch tab on the docker image page: https://github.com/ep1cman/unifi-protect-backup/pkgs/container/unifi-protect-backup
Arm64 docker images are published. If you are unable to pull this, please upgrade to a 64-bit OS. 32-bit docker images are no longer published for this project.
